5_vision_language_models/vlm_finetuning.md

```diff
@@ -0,0 +1,5 @@
+# VLM Fine-Tuning
+
```

Comment:
We should start with a section on the data representation of conversation in messages with images. It can basically be a brief version of [this](https://github.com/huggingface/smol-course/blob/main/1_instruction_tuning/chat_templates.md). And it can explain the `format_data` function from [here](https://huggingface.co/learn/cookbook/fine_tuning_vlm_trl)